src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1841,12 +1845,16 @@ public static bool IsConstantNumericZero(ConstantValue value)
                     return value.Int32Value == 0;
                 case ConstantValueTypeDiscriminator.Int64:
                     return value.Int64Value == 0;
+                case ConstantValueTypeDiscriminator.NInt:
```

Comment:
Should we add labels to the Int32 and UInt32 sections instead of creating new switch sections for these?